modules/cloudbuild/cloudbuild_builder/Dockerfile

```diff
@@ -32,7 +32,18 @@ RUN apt-get update && \
     sha256sum -c terraform_SHA256SUMS --status && \
     unzip terraform_linux_amd64.zip -d /builder/terraform && \
     rm -f terraform_linux_amd64.zip && \
-    gsutil cp gs://terraform-validator/releases/${ENV_TERRAFORM_VALIDATOR_RELEASE}/terraform-validator-linux-amd64 /builder/terraform/terraform-validator && \
+    VALIDATOR_RELEASE_VERSION=$(echo $ENV_TERRAFORM_VALIDATOR_RELEASE | sed -e 's/^v//') && \
+    VALIDATOR_RELEASE_NUMBER_AS_INT=$(echo $VALIDATOR_RELEASE_VERSION | sed -e 's/\.//g;s/^0//') && \
+    TARBALL_VERSION_THRESHOLD_AS_INT="60" && \
+    GCS_BASE_PATH="gs://terraform-validator/releases/${ENV_TERRAFORM_VALIDATOR_RELEASE}" && \
+    LEGACY_GCS_PATH="${GCS_BASE_PATH}/terraform-validator-linux-amd64" && \
+    CURRENT_GCS_PATH="${GCS_BASE_PATH}/terraform-validator_linux_amd64-${VALIDATOR_RELEASE_VERSION}.tar.gz" && \
+    if ( echo "$ENV_TERRAFORM_VALIDATOR_RELEASE" | grep -v '-' | grep -q '^v' ) && \
```

Comment:
I think we should drop support for older TFV with a breaking change rather than have this conditional logic. We can also have a[ custom validation rule ](https://www.terraform.io/docs/language/values/variables.html#custom-validation-rules)to support this and surface an actionable error.

Reply:
Hi @bharathkkb - thanks for your feedback.

I support your suggestion to opt for a breaking change in order to remove the need for extra conditional logic. Adding a custom validation rule to ensure that the TFV version is >= v0.6.0 would also be a useful measure.

On that basis I will rework this PR to reflect those changes.